### DISCUSSION TITLE
Text of pinned profile cards is cropped

### DISCUSSION THREAD

**Discussion by nohli** [community/community#43527] [Mobile]
When keeping the app open in the background, on the profile page, opening again after some time and pulling down to refresh, the height of pinned cards shrinks.
This makes the letters of a repo's description get cropped vertically.
One whole line is missing, plus the bottom part of the letters is cropped.

It seems hard to reproduce, but definitely happens sometimes (see comments).

Issue:
![image](https://user-images.githubusercontent.com/43643339/211113981-29bedab0-3ea0-4a4d-8830-59e419d23944.png)

Normal:
![image](https://user-images.githubusercontent.com/43643339/214996309-471a5f96-845f-49fd-9fb5-d4b861c3f4ee.png)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.94.0 (86495482)
Device: iPhone 12 mini (iOS 16.2)
</details>

**Comment by Iron-Ham** [comment]
@nohli Thank you for reaching out! I've been trying to reproduce this locally for some time and have been unable to. Are you still running into this, and if so: is there any specific guideline to help reproduce this? 


Thanks!

**Reply by nohli** [reply]
Thanks for working on this issue 😊

I can't reproduce it immediately.

But many times, when the app is already open for one day, then coming from the background, and I pull-to-refresh the profile page, the cards collapse a bit and this issue occurs.

So I'd say try to reproduce it like this:
1. Open the GitHub iOS app
2. Leave it in the background for one day (and use other apps)
3. Open app
4. PTR on the profile page
5. See if the letters in last row of the description are cropped at the bottom

**Reply by Agus110500** [reply]
N

**Comment by nohli** [comment]
Today, the bug is worse (went from iPhone Mini to Pro - could be because of the different size)

![image](https://user-images.githubusercontent.com/43643339/222988903-434b144e-4b19-421b-8895-ef057354eb38.png)

**Comment by Bobbert23** [comment]
> When keeping the app open in the background, on the profile page, opening again after some time and pulling down to refresh, the height of pinned cards shrinks.
> This makes the letters of a repo's description get cropped vertically.
> One whole line is missing, plus the bottom part of the letters is cropped.
> 
> It seems hard to reproduce, but definitely happens sometimes (see comments).
> 
> Issue:
> ![image](https://user-images.githubusercontent.com/43643339/211113981-29bedab0-3ea0-4a4d-8830-59e419d23944.png)
> 
> Normal:
> ![image](https://user-images.githubusercontent.com/43643339/214996309-471a5f96-845f-49fd-9fb5-d4b861c3f4ee.png)
> 
> <details>
> <summary>ℹ️ Additional Debug Info</summary>
> GitHub for iOS, Version 1.94.0 (86495482)
> Device: iPhone 12 mini (iOS 16.2)
> </details>

**Comment by nohli** [comment]
It's still an issue

![image](https://github.com/community/community/assets/43643339/0ab303e7-014a-419a-a420-e84b3acbf59c)
